content/SCALE/SCALECLIReference/Account/CLIuser.md

```diff
@@ -476,56 +439,50 @@ account user set_attribute id=<i>3001</i> key="<i>favorite_color</i>" value="<i>
-This means that you have either performed a fresh install of SCALE, and chose to not configure an admin or root password in the TrueNAS Console installer, or you have reset configuration to defaults and have not yet logged in to the SCALE Web UI.
+The `setup_local_administrator` command only works if both a local administrator and root password are not configured.
+This means that you have either performed a fresh install of SCALE using option 3 to configure using the Web UI and not configured an admin or root password, or you reset the configuration to defaults and have not yet logged in to the SCALE Web UI.
 If either a local administrator or a root password exist, this command returns `Error: Local administrator is already set up.`
 {{< /hint >}}
 
-{{< expand "Using the Setup_Local_Administrator Command" "v" >}}
+{{< include file="CLICommandWIP.md" type="page" >}}
 
+{{< expand "Using the Setup_Local_Administrator Command" "v" >}}
 #### Description
-
-Enter the CLI from the [Console setup menu]({{< relref "ConsoleSetupMenuScale.md" >}}) before configuring a local administrator or root user account.
-
+The `setup_local_administrator` command has two required properties, `username`, `password`. 
+The `options` property is not required and is entered as an array.
+Enter property arguments using the `=` delimiter to separate the property and value, then enter <kbd>Enter</kbd>.
 The command returns a blank line.
+<!-- Commenting this out until it is verified in cobia.
+Enter the CLI from the [Console setup menu]({{< relref "ConsoleSetupMenuScale.md" >}}) before configuring a local administrator or root user account. -->
+{{< hint type=note >}}
+`setup_local_administrator` accepts only "admin" or "root" for **username**.
+It is recommended to use "admin." Root user access is a deprecated method.
+{{< /hint >}}
 
 To confirm the administrator is set up, use [`has_local_administrator_set_up`](#has_local_administrator_set_up-command), [`get_user_obj`](#get_user_obj-command), or log in to the SCALE Web UI using the admin username and password.
 
 #### Usage
 
 From the CLI prompt, enter:
 
-<code>account user setup_local_administrator username:admin password:<i>passw0rt</i></code>
-
-Press <kbd>Enter</kbd>
-
-From the **account** prompt, enter:
-
-<code>user setup_local_administrator username:admin password:<i>passw0rt</i></code>
-
-Press <kbd>Enter</kbd>
+<code>account user setup_local_administrator username:<i>admin</i> password:<i>passw0rt</i></code>
 
-Where *passw0rt* is the desired admin account password.
-
-{{< hint type=note >}}
-`setup_local_administrator` accepts only "admin" or "root" for **username**.
-It is recommended to use "admin." Root user access is a deprecated method.
-{{< /hint >}}
+Where:
+* *passw0rt* is the password for the admin account.
+* *admin* is the administration user. If the `username` value is *root* it sets up the root user.
```

Comment:
I think the deleted hint, or a version of it, should still be here. My intention there was to explain that the command will fail unless username is either "admin" or "root". You can't create an administrator named anything else using this command

Reply:
details like a hint about entering a command belong in the Description section not in the Usage where we just demonstrate entering the command. I covered this in the description. the reason admin is a variable is because you can enter one or the other which makes it a variable. If neither the admin or root user are created then you can enter either one. Again, details about this belong in the Description text not in the Usage.,